src/coreclr/tests/src/JIT/HardwareIntrinsics/Arm/AdvSimd/BitwiseSelect.Vector128.Double.cs

```diff
@@ -546,15 +546,15 @@ private void ValidateResult(Double[] firstOp, Double[] secondOp, Double[] thirdO
         {
             bool succeeded = true;
 
-            if (Helpers.BitwiseSelect(firstOp[0], secondOp[0], thirdOp[0]) != result[0])
+            if (BitConverter.DoubleToInt64Bits(Helpers.BitwiseSelect(firstOp[0], secondOp[0], thirdOp[0])) != BitConverter.DoubleToInt64Bits(result[0]))
```

Comment:
Why the need for the conversion?

Reply:
I forgot to add this when I was implementing BitwiseSelect - the reason why it's needed is that we can't use double/single comparison since the output of BitwiseSelect can be NaN or INF and we want to test the actual and expected value on bitwise equality.

Reply:
Ah, of course - thanks!